src/components/Header/Nav/ProductNavigation/Desktop/MegaMenu.tsx

```diff
@@ -178,94 +166,77 @@ export const megaMenuSections = {
   },
 }
 
-function MegaMenu({ cancel, id }: MegaMenuProps) {
-  useEffect(() => {
-    const onESC = (ev: KeyboardEvent) => {
-      if (ev.key === "Escape") {
-        cancel()
-      }
-    }
-    window.addEventListener("keyup", onESC, false)
-    return () => {
-      window.removeEventListener("keyup", onESC, false)
-    }
-  }, [])
-
+function MegaMenu({ cancel }: { cancel?: () => void }) {
   return (
-    <div className={styles.megaMenuContainer} id={id}>
-      <div className={styles.wrapper}>
-        <div className={styles.column}>
-          <div className={styles.section}>
-            <header>
-              <BlueSquare />
-              <GroupTitle>{megaMenuSections.orchestration.title}</GroupTitle>
-            </header>
-            <div className={styles.itemList}>
-              {megaMenuSections.orchestration.items.map((link) => (
-                <GroupItem key={link.title} data={link} />
-              ))}
-            </div>
-          </div>
-          <div className={styles.section}>
-            <header>
-              <BlueSquare />
-              <GroupTitle>{megaMenuSections.data.title}</GroupTitle>
-            </header>
-            <div className={styles.itemList}>
-              {megaMenuSections.data.items.map((link) => (
-                <GroupItem key={link.title} data={link} />
-              ))}
-            </div>
+    <div className={styles.wrapper} onMouseLeave={cancel}>
+      <div className={styles.row}>
+        <div className={styles.section}>
+          <header>
+            <BlueSquare />
+            <GroupTitle>{megaMenuSections.orchestration.title}</GroupTitle>
+          </header>
+          <div className={styles.itemList}>
```

Comment:
All these items should not be divs but UL. The actual change would require the full menu to be changed to a OL, UL, but that is too m,uch refactoring and not needed by CL. Just change list items to a list will suffice for A11Y needs